app/models/yardsale_tag.rb

```diff
@@ -0,0 +1,4 @@
+class YardsaleTag < ApplicationRecord
+    has_many: yard_sales
+    has_many: tags
```

Comment:
Should a single YardSale tag represent the connection between a Yard Sale + Tag? I don't think this should have many tags, because each record represents a single tag linked to a single yard sale. This should be belongs_to for both

Reply:
shoot, yeas you are right error on my end. It make sense. Making the changes